lib/arqo/query.rb

```diff
@@ -0,0 +1,32 @@
+# frozen_string_literal: true
+
+module ARQO
+  # Parent class for query objects
+  class Query
+    attr_reader :relation
+    delegate_missing_to :relation
```

Comment:
Is there a way we can only delegate methods that are defined in the scope module?

Reply:
I guess so, but we'd need hacky code around `method_missing`. What issues do you see with this implementation?

Reply:
None, I would just prefer to make it explicit. But I don't see any issue with current approach